services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_compose_specs.py

```diff
@@ -301,7 +299,7 @@ def assemble_spec(
     # TODO: will be used in next PR
     assert product_name  # nosec
 
-    stringified_service_spec = replace_env_vars_in_compose_spec(
+    stringified_service_spec: str = replace_env_vars_in_compose_spec(
```

Comment:
I guess the TODO above is outdated right?

Reply:
it's from two months ago. I guess there is some work in progress here